examples/azure-resource-lab-chart/templates/cluster.yaml

```diff
@@ -1,5 +1,5 @@
-apiVersion: "giantswarm.io/v1"
-kind: AzureCluster
+apiVersion: "provider.giantswarm.io/v1alpha1"
+kind: AzureConfig
```

Comment:
OK in this case as its small. But in general upgrading the chart from TPR to CRD would be better as a separate PR since its a separate change.